### DISCUSSION TITLE
The green (Beta) part of the For You "tab" looks like an unread state

### DISCUSSION THREAD

**Discussion by wez** [community/community#13305] [Other Feature Feedback, Questions, & Ideas]
I keep clicking on it to try to make it stop being highlighted/lit up, but then I'm disappointed because it shows me stuff I don't want to see and still doesn't dispel the green bit.

I don't want an algorithmic feed; I don't use GH to passively explore/find other repos, I use it to keep an eye on my projects and the projects that I'm following: I care about releases in those projects and state changes/stars on my repos.  I don't really care very much about what other people are doing on GH, and I certainly don't want suggested content to be injected--I truly hate the equivalent functionality on twitter, YT and FB and I really don't wish to see it here on GH.

Please let me get rid of the green "(Beta)" bits so that they are less OCD-triggering: I'd suggest moving the beta indication inside the tab itself rather than in the label, and then I can happily ignore the product feature.

**Comment by sntwandy** [comment]
It's just a badge indicating that it's a beta, it will only be there for a moment and then it won't be, but maybe take it into consideration.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
